tests/functional/spec/features/hub_and_spoke/choose_another_section.spec.js

```diff
@@ -0,0 +1,56 @@
+const helpers = require('../../../helpers');
+const EmploymentStatusBlockPage = require('../../../generated_pages/hub_and_spoke/employment-status.page.js');
+const ProxyPage = require('../../../generated_pages/hub_and_spoke/proxy.page.js');
+const HubPage = require('../../../base_pages/hub.page.js');
+
+describe('Choose another section link', function () {
+
+  it('When a user first views the Hub, then the link should not be displayed', function () {
+    return helpers.openQuestionnaire('test_hub_and_spoke.json')
+      .then(() => {
+        return browser
+          .getText('body').should.not.eventually.contain('Can\'t complete this question?')
```

Comment:
The tests shouldn't be accessing the raw body content where possible, instead, we should introduce a method in `tests/functional/base_pages/question.page.js` to get the attribute for a return to hub link.

Reply:
I created a hub link method in question.pages.js and use it in tests now